fachhochschule-vorarlberg-author-date.csl

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
   <info>
-    <title>Fachhochschule Vorarlberg (author-date, German)</title>
+    <title>Fachhochschule Vorarlberg (author-date, German, English)</title>
```

Comment:
same question/comment about language as for the note style

Reply:
Same answer/comment, i have no objection about it.